pkg/framework/operators/baseoperator.go

```diff
@@ -334,6 +337,27 @@ func (b *BaseOperator) setupDeployment(jsonItem []byte) {
 	}
 }
 
+func (b *BaseOperator) getClientImage() string {
+    if (b.clientImageName!="") {
+        return b.clientImageName
+    } else {
+        return ClientImageName
+    }
+}
+
+func (b *BaseOperator) setupClients() {
+    log.Logf("Setting up clients image")
+    clientsDeploymentConfig := &appsv1.Deployment{}
+    clientsDeploymentConfig.Spec.Template.Spec.Containers[0].Name="systemtests-clients"
+    clientsDeploymentConfig.Spec.Template.Spec.Containers[0].Image=ClientImageName
+    clientsDeploymentConfig.Spec.Template.Spec.Containers[0].Command[0]="sleep"
+    clientsDeploymentConfig.Spec.Template.Spec.Containers[0].Args[0]="infinity"
+    clientsDeploymentConfig.Spec.Template.Spec.Containers[0].ImagePullPolicy="Always"
+    if _, err := b.kubeClient.AppsV1().Deployments(b.namespace).Create(clientsDeploymentConfig); err != nil {
+		b.errorItemCreate("deployment", err)
+	}
+}
+
```

Comment:
same as the others... not part of the baseoperator to me.

Reply:
Alright, I'll move it out. Thanks!